MaxSukhanov/trainee-capybara_scraper
# first iteration of simple onliner scraper

What's new here? 

First working version of simple scraper built with Capybara, Nokogiri and SeleniumWebDriver

How to test this stuff?
0. Check if you have geckodriver to use that with Firefox like I do (if not get it here: https://github.com/mozilla/geckodriver/releases)
1. Go to the root directory 
2. Run `scraper` executable
3. Wait for a while (this stuff is quite slow, so grab some coffee)
4. Check `test.csv` to find the latest news from all news sections of Onliner.by
